pkgs/by-name/wh/whatsapp-chat-exporter/package.nix

```diff
@@ -4,16 +4,19 @@
   fetchFromGitHub,
 }:
 
-python3Packages.buildPythonApplication rec {
+let
+  version = "0.12.0";
```

Comment:
You don't necessarily have to avoid using recursive (This recommendation is not binding).